lib/core/module.js

```diff
@@ -71,6 +71,12 @@ export default class ModuleContainer {
 
     // Add to nuxt layouts
     this.options.layouts[name || path.parse(src).name] = `./${dst}`
+
+    // If error layout, set ErrorPage
+    if (name === 'error') {
```

Comment:
Can we add it as another function like `setErrorLayout`? This is somehow a hidden behaviour

Reply:
Done :relaxed: